### PR TITLE
Added Chrome profile detection (closes #29)

### LINKED ISSUE #29: List chrome profiles as separate options
originally requested through email.

to launch:
```
chrome --profile-directory={profile} {url}
```
to discover: (until i find a better way)
```
glob %localappdata%\google\chrome\*\Google Profile.ico
```
folder name is profile name, and the `.ico` file is the icon

### PR DESCRIPTION
**Pull request by kueswol**
Added functionality to add Chrome profiles.

If Chrome browser is detected, we're searching for profiles. If any are found, we'll add the Chrome instances with the profiles and remove the default one.

### CODE REVIEW

File: BrowserSelect/Program.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ namespace BrowserSelect
 {
     static class Program
     {
-        public static string url = "http://google.com/";
+        public static string url = "https://www.golem.de/";
```
Comment (zumoshi) [inline BrowserSelect/Program.cs]: any particular reason for changing this?

File: BrowserSelect/Form1.cs [hunk 2]
```diff
@@ -201,7 +201,9 @@ private void browser_click(object sender, EventArgs e)
 
         public static void open_url(Browser b, bool incognito = false)
         {
-            var args = new List<String>();
+            var args = new List<string>();
+            if (b.exec.ToLower().EndsWith("chrome.exe") && !String.IsNullOrEmpty(b.additionalArgs))
```
Comment (zumoshi) [inline BrowserSelect/Form1.cs]: why check if it's chrome? always appending additional args if not null or empty doesn't seem to cause any side effects and could be useful in the future.

**Comment by zumoshi** [comment]
hi,
thanks for your work, just tested it. works nicely.
there are just a few minor issues before I could merge it, which I can fix myself if you don't have the time.
1. you didn't include `Newtonsoft.Json.dll` in `installer.nsi`, which means it won't be included in setup, and therefore the app won't run complaining about missing dll
2. shortcuts for profiles are messed up. with `2`, `c`, and `(` opening chrome... it would be nice if the first letter of the profile's name was the third shortcut instead of open parenthesis.
3. version needs to be bumped to 1.4.0
4. if we're releasing a new version might as well fix #35 

also would be nice if you could squash the commits so it's a single commit, and check the comments I made on the diff.

p.s. my first pull request! yay! the first time someone made a pull request on one of my projects

File: BrowserSelect/Program.cs (hunk 1 above)
Comment (kueswol) [reply BrowserSelect/Program.cs]: damn, sorry, forgot about this. Was trying to figure out your nice project and tested quite a few different domains here - just forgot to refert it back,

**Comment by kueswol** [comment]
I'll fix the stuff you've mentioned. Regarding you points:

1. The dependency should be added by NuGet. I think when packaging the project NuGet should take care of it, doesn't it?
2. Shortcuts will be fixed.
3. I'll surely figure this out how to do it.
4. I've never used Opera - haven't installed it either. If you won't mind, I'll let you take care of it.

yeah, the commit stuff was a bit messed up. First time actually working with GIT ;-)

p.s. congratulations! Don't worry, it's also my first time

File: BrowserSelect/Form1.cs (hunk 2 above)
Comment (kueswol) [reply BrowserSelect/Form1.cs]: didn't want to mess up things. But you're completely right. As the string is always initialized at least with an empty string, this should make no difference.

**Comment by zumoshi** [comment]
1. nope, nuget automatically downloads the dll when you run the app in visual studio, but unless we ask users to also use visual studio, nuget won't do that. I'm using NSIS to create the setup which is a 3rd party app unrelated to visual studio and nuget.
3. right click project name -> properties -> assembly information.
4. ok.

thanks again for what you've done so far.

**Comment by kueswol** [comment]
if I figured it out correctly, I'm supposed to do the changes with a new Pull Request, which means, we may close this one.

**Comment by zumoshi** [comment]
I'm pretty sure you could've just added the changes to this pull request.
https://stackoverflow.com/a/9790530/1461004

but you've already made another one, oh well.

**Pull request closed without merging**